app/Http/Controllers/AdminController.php

```diff
@@ -20,4 +24,16 @@ public function showUsers()
         $all_users = $this->user->all();
         return view('admin.users.allusers')->with('all_users',$all_users);
     }
```

Comment:
Why there is a closing curly braces here?
Did your VS code give you an error indicator about this line?